boilerplate/update.cfg

```diff
@@ -5,3 +5,4 @@ lyft/pull_request_template
 lyft/github_workflows
 lyft/golang_dockerfile
 lyft/docker_build
+flyte/
```

Comment:
May be not this PR but we can have an issue created to remove the lyft references

Reply:
Oh. Let me merge this one, and if we do not need the `lyft` references, we can remove them later.